BowlingBall.Tests/GameFixture.cs

```diff
@@ -5,10 +5,146 @@ namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        private Game game;
+        
+        //Writing the inilialization in the Constructor as there is no TestInilializer
+
+        //First fail both the methods in the Game class
+        
+        public GameFixture(){
+            game = new Game();  
+        }
+
         [Fact]
         public void DummyTest()
         {
             // This is a dummy test that will always pass.
         }
+
+        [Fact]
+        public void DoesGameClassExists(){
+            Assert.NotNull(this.game);
+        }
+
+        public void RollMany(int pins, int rolls){
+
+            for(int i = 0; i < rolls; i++){
+                this.game.Roll(pins);
+            }
+        }
+
+        [Fact]
+
+        public void canRollAllZerosGame(){
+            
+            RollMany(0,20);
+            int expectedScore = 0;
+            int actualScore = this.game.GetScore();
+            Assert.Equal(expectedScore,actualScore);
+        }
+
+        [Fact]
+        public void canRollAllTensGame(){

```

Comment:
All function names need to be in all CAPS

Reply:
Ok sir